documentation/performance/sizing-search.html

```diff
@@ -1,705 +1,581 @@
+we configure depends on what kind of availability guarantees the deployment should provide.  
+Vespa allows configuring the availability versus cost using the 
+<em>redundancy</em> and <em>searchable-copies</em> settings, described below: </p>
+
+<table class="table">
+  <thead></thead><tbody>
+    <tr id="redundancy">
+      <th>Redundancy</th>
+      <td>
+        <p>
+        <a href="../reference/services-content.html#redundancy">Redundancy</a>
+        Defines the total number of copies of each piece of data the cluster will store and maintain to avoid data loss. 
+        Example: with a redundancy of 2, the system tolerates 1 node failure before data becomes unavailable. 
+        </p>
+      </td>
+    </tr><tr id="searchable-copies">
+      <th>Searchable copies</th>
+      <td>
+        <p>
+        <a href="../reference/services-content.html#searchable-copies">Searchable-copies</a>
+        configures how many of the copies (as configured with <em>redundancy</em>) to be indexed 
+        (<em>ready</em>) at any time. Configuring
+        <em>searchable-copies</em> to be less than redundancy saves resources (memory, disk, cpu) 
+        as not all copies are indexed (<em>ready</em>).  In case 
+        of node failure the remaining nodes needs to activate (index) the 
+        <em>not ready</em> documents which belonged to the failed node. In this transition
+        period the search is operating with reduced search coverage. 
+        </p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+<figure>
+<figcaption><b>Content node database illustration</b></figcaption>
+<img src="../img/proton_database.svg"  height="420" width="840"/>
+</figure>
+<p>
+The above figure illustrates the three 
+<a href="/documentation/proton.html#sub-databases">sub-database</a> inside a Vespa content node. </p>
+
+<ul>
+<li>The documents in the <b>Ready</b> DB is indexed but only the documents in <b>Active</b> state is searchable. 
+In a flat distributed system there is only one active 
+instance of the same document, while with grouped distribution there is one active instance per group.</li>
+
+<li>The documents in the <b>Not Ready</b> DB is stored but not indexed</li>
+
+<li>The documents in the <b>Deleted</b> are stored but blacklisted so they are hidden from search until 
+permanently deleted
+from storage by <a href="proton.html#proton-maintenance-jobs">Proton maintenance jobs</a></li>
+
+</ul>
+<p>If the availability guarantees tolerate temporary search coverage loss during node 
+failures using <em>searchable-copies</em> 1 is by far the 
+most optimal for for serving performance as query work per node becomes
+less as index structures like posting lists are smaller as they then only contains documents in <em>Active</em> 
+state and not also the <em>Not Active</em> documents.</p>
+With <em>searchable-copies</em> 2 and <em>redundancy</em> 2 each content node will ready index both replicas 
```

Comment:
"each content node" may be a bit confusing here, consider perhaps rephrasing as `With <em>searchable-copies</em> 2 and <em>redundancy</em> 2 each replica is fully indexed on separate content nodes (...)`